js/renovation/ui/pager/pages/__tests__/page.test.tsx

```diff
@@ -20,6 +20,24 @@ describe('Small pager pages', () => {
     });
   });
 
+  // T1109686
+  it('pageIndexes: check aria-current attribute', () => {
```

Comment:
Split this test into two parts. One should test page and another should test light button